pdns/recursordist/rec-main.cc

```diff
@@ -2554,6 +2556,11 @@ static void houseKeepingWork(Logr::log_t log)
       SyncRes::pruneSaveParentsNSSets(now.tv_sec);
     });
 
+    static PeriodicTask pruneCookiesTask{"pruneCookiesTask", 30};
+    pruneCookiesTask.runIfDue(now, [now]() {
+      pruneCookies(now.tv_sec - 1800);
```

Comment:
RFC 9018 section 4.3 says that cookies up to an hour old can be used. I'd recommend using 3000 seconds here (50 minutes).